### ISSUE TITLE
Cannot select text in input field

### ISSUE THREAD

**Issue by alyrik** [Popmotion/popmotion#95]
This happens after some animations play. It is still possible to select input text via dblclick but it's impossible to do this by dragging a mouse pointer. Form is inside a tracked and animated element.
Is there a way to remove popmotion instance from a connected element?

CodePen demo: http://codepen.io/anon/pen/OygYgB

**Comment by InventingWithMonster** [comment]
Hey man

The problem here is probably `preventDefault` - the default being the input gains focus. You can probably comment both these things out and it'll work properly:

        $body.on('mousedown touchstart', ballActor.element, function (event) {
            //event.stopPropagation();
            //event.preventDefault();

            ballActor.start(trackBall, event);

            $body.on('mouseup touchend', function () {
                ballActor.stop();
            });
        });

**Comment by alyrik** [comment]
It doesn't help. Even if I remove this handler after mouseup:

```
$body.on('mouseup touchend', function () {
    $body.off('mousedown touchstart mouseup touchend');
    ballActor.stop();
});
```

The question is how to completely deactivate popmotion attached to a specific element?

**Comment by InventingWithMonster** [comment]
Well Popmotion isn't actually attached to a specific element at any point - that's all down to whichever method you choose to attach event handlers.

In this instance,

**Comment by InventingWithMonster** [comment]
I see what you mean now, I'll look into this further. I have an idea what's causing this.

**Comment by InventingWithMonster** [comment]
@alyrik Hey dude I've just pushed a fix with 3.3.2, could you let me know if it works for you? Thanks

**Comment by alyrik** [comment]
Yeah, it works here - [CodePen example](http://codepen.io/anon/pen/OygYgB). Thank you! Great!

Though it doesn't work in my project but it may be my fault. Trying to overcome.
